docs/rules.md

```diff
@@ -113,7 +114,7 @@ The following **97** rules are applied by this linter:
 | E7011 <a name="E7011"></a> | Mapping name limit not exceeded | Check the size of Mapping names in the template is less than the upper limit | [Source](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cloudformation-limits.html) | `mappings`,`limits` |
 | E7012 <a name="E7012"></a> | Mapping attribute limit not exceeded | Check if the amount of Mapping attributes in the template is less than the upper limit | [Source](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cloudformation-limits.html) | `mappings`,`limits` |
 | E8001 <a name="E8001"></a> | Conditions have appropriate properties | Check if Conditions are properly configured | [Source](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/conditions-section-structure.html) | `conditions` |
-| I3011 <a name="I3011"></a> | Check DynamoDB tables have a set DeletionPolicy | The default action when removing a DynamoDB Table is to delete it. This check requires you to specifically set a DeletionPolicy and you know the risks | [Source](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-attribute-deletionpolicy.html) | `resources`,`dynamodb` |
```

Comment:
Make sure you run the documentation creator with include_checks I

Reply:
Wut... Oh, can I make that the default behaviour?

Reply:
Of course but it would take a little work. The config pull request may be required first.

Right now it spits out the documentation based on the loaded rules. So we can modify that behavior to document all rules no matter what. Basically we would overwrite the included and ignore checks when doing a documentation dump so it covers everything.
You have a cfnlintrc config file that would always load all the rules.